src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10234,7 +10234,7 @@ namespace UnitarySystems {
             } else {
                 QZnReq =
                     state.dataZoneEnergyDemand->ZoneSysEnergyDemand(this->ControlZoneNum).RemainingOutputRequired / this->ControlZoneMassFlowFrac;
-                if (this->m_StageNum > 0) {
+                if (QZnReq > 0.0) {
```

Comment:
Load sign determines cooling and heating flags

Reply:
Yeah that makes sense.  But this does imply that a zero load will set `.CoolingLoad` to true.  Is that what you want?  Or should there be a check for `abs(QZnReq) < SomeTolerance` to set both load flags to false?

Reply:
The thermostat is sending the m_StageNum status just above here at line 10216 so this looks OK.

    this->m_StageNum = state.dataZoneEnergyDemand->ZoneSysEnergyDemand(this->ControlZoneNum).StageNum;

Reply:
Thanks @rraustad, OK no reason to hold this up.